client/bin/build_package.ts

```diff
@@ -1,10 +1,13 @@
-import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
+import { build, emptyDir } from "https://deno.land/x/dnt@0.25.1/mod.ts";
 
 await emptyDir("./dist");
 
 await build({
   entryPoints: ["client/src/index.ts"],
   outDir: "client/dist",
+  compilerOptions: {
+    lib: ["es2021", "dom"],
+  },
```

Comment:
`dnt` wasn't pinned. It had a breaking change which requires passing explicit options to the typescript compiler. Without these options, the build fails when it tries to reference DOM APIs.